README.md

```diff
@@ -13,3 +13,31 @@ and therefore following our [license](./LICENSE) (MPL-2.0)
 do not necessarily carry a Source Code Form License Notice.
 The Source Code Form for each such file is available from
 [mozilla/mozilla-firefox/firefox](https://github.com/mozilla-firefox/firefox).
+
+## Adding a New Version
+
+Before adding a new version, make sure to merge pending update pull requests
+(it will avoid the need to rebase existing PRs). Then:
+
+1. Create a local branch.
+2. Add the new version to the update configuration in `.github/update-config.json`.
+Note that ESR versions should be ordered by the most recent to the least recent
+(e.g. `esr140` before `esr128`). The name of the version needs to match the
+branch name on [mozilla/mozilla-firefox/firefox](https://github.com/mozilla-firefox/firefox).
+5. Open a pull request.
+6. Once the pull request is merged, run the update automation. This will generate
+the corresponding data storage file in `_data`.
+
+## Removing a Supported Version
+
+Before removing a supported version from the repository, make sure to merge
+pending update pull requests and pause sync in Pontoon. Then:
```

Comment:
Should the sync from the merge run before pausing sync?

Reply:
It's not strictly necessary, since it's only removing content. Pontoon will pick up the removal once the removal PR is merged.